pages/en-us/components/input.mdx

```diff
@@ -330,12 +311,10 @@ type InputSizes = 'mini' | 'small' | 'medium' | 'large'
 type InputColor = 'default' | 'secondary' | 'success' | 'warning' | 'error'
 ```
 
-<Attributes.Title>useInput</Attributes.Title>
+<Attributes.Title>useInputHandle</Attributes.Title>
 
 ```ts
-type useInput = (
-  initialValue: string,
-) => {
+type useInputHandle = () => {
```

Comment:
please check the useInputHandle api document

Reply:
done